README.md

```diff
@@ -11,7 +11,7 @@ for details on getting up and running with your very own robot friend.
 In most cases, you'll probably never have to hack on this repo directly if you
 are building your own bot. But if you do, check out [CONTRIBUTING.md](CONTRIBUTING.md)
 
-If you'd like to chat with Hubot users and developers, drop by #hubot on FreeNode IRC.
+If you'd like to chat with Hubot users and developers, [join us on Slack](https://hubot-slackin.herokuapp.com/).
```

Comment:
Slack has official IRC support. Are ther advantages to using Freenode over that?

My preference would be to reduce the surface area here and use one service.